lib/user/pages/user_page.dart

```diff
@@ -68,9 +70,19 @@ class _UserPageState extends State<UserPage> {
               )
             : null,
         actions: [
+          Padding(
+            padding: const EdgeInsets.fromLTRB(0.0, 4.0, 0.0, 4.0),
+            child: IconButton(
+              onPressed: () => userBloc?.add(ResetUserEvent()),
+              icon: const Icon(
+                Icons.refresh_rounded,
+                semanticLabel: 'Refresh',
```

Comment:
We can now use the localized strings for this! There should be examples of how to do this with some of the other files